test/build-tasks-sample/src/contract.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <NeoContractName>$(AssemblyName)</NeoContractName>
+    <NeoExpressBatchFile>../express.batch</NeoExpressBatchFile>
+    <Nullable>enable</Nullable>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
after tools is updated, can you change this to `dotnet` 7.0 or you can create a `Directory.Build.props` file in the test folder?